recipes/libjpeg/all/conanfile.py

```diff
@@ -109,14 +128,33 @@ def _build_nmake(self):
                 replace_in_file(self, "Win32.Mak", "rc     = Rc", f"rc     = {rc}")
             # set flags directly in makefile.vc
             # cflags are critical for the library. ldflags and ldlibs are only for binaries
-            if is_msvc_static_runtime(self):
-                replace_in_file(self, "makefile.vc", "(cvars)", "(cvarsmt)")
-                replace_in_file(self, "makefile.vc", "(conlibs)", "(conlibsmt)")
+            if is_msvc(self):
+                msbuild = MSBuild(self)
+                if is_msvc_static_runtime(self):
+                    replace_in_file(self, "makefile.vs", "(cvars)", "(cvarsmt)")
```

Comment:
the runtime logic is already handled by the `.props` file generated by Conan  based on the settings. :)